packages/vue2/package.json

```diff
@@ -26,9 +26,9 @@
 	"exports": {
 		"./*": "./*",
 		".": {
-			"import": "./dist/index.mjs",
-			"types": "./dist/index.d.ts",
-			"default": "./dist/index.js"
+			"import": "./dist/iconifg.mjs",
```

Comment:
Typo: 'iconifg' ('g' at the end)

Reply:
Sorry about that. I wrote it so fast...

Reply:
No worries. Could be worse, could have written 'index' instead of 'iconify'... :)